store/tikv/latch/latch_test.go

```diff
@@ -86,88 +86,3 @@ func (s *testLatchSuite) TestWakeUp(c *C) {
 	c.Assert(acquired, IsTrue)
 	c.Assert(stale, IsFalse)
 }
-
-type txn struct {
-	keys    [][]byte
-	startTS uint64
-	lock    Lock
-}
-
-func newTxn(keys [][]byte, startTS uint64, lock Lock) txn {
-	return txn{
-		keys:    keys,
-		startTS: startTS,
-		lock:    lock,
-	}
-}
-
-type txnScheduler struct {
-	txns    map[uint64]*txn
-	latches *Latches
-	lock    sync.Mutex
-	wait    *sync.WaitGroup
-}
-
-func newTxnScheduler(wait *sync.WaitGroup, latches *Latches) *txnScheduler {
-	return &txnScheduler{
-		txns:    make(map[uint64]*txn),
-		latches: latches,
-		wait:    wait,
-	}
-}
-
-func (store *txnScheduler) runTxn(startTS uint64) {
-	store.lock.Lock()
-	txn, ok := store.txns[startTS]
-	store.lock.Unlock()
-	if !ok {
-		panic(startTS)
-	}
-	acquired, stale := store.latches.Acquire(&txn.lock)
```

Comment:
Two different goroutine call Acquire a same lock is the cause of DATA RACE, but the root problem,
this code is too vulnerable and bug prone. I just delete it.